README.md

```diff
@@ -41,11 +41,15 @@ The easiest way to get etcd is to use one of the pre-built release binaries whic
 
 For those wanting to try the very latest version, you can build the latest version of etcd from the `master` branch.
 You will first need [*Go*](https://golang.org/) installed on your machine (version 1.6+ is required).
+etcd vendors its dependency for official release binaries, while making vendoring optional to avoid import conflicts.
```

Comment:
move this to https://github.com/coreos/etcd/blob/master/Documentation/dl_build.md#build-the-latest-version?

Also put a link here to the dl_build.md page instead?